README.md

```diff
@@ -40,22 +30,27 @@ Debian and derived distributions would be:
 sudo apt install libxml2-dev
 ```
 
-#### From the source code
+## Usage
 
-```bash
-git clone https://github.com/bblfsh/client-python.git
-cd client-python
-make install
-```
+A small example of how parse a Python file and extract the import declarations from the [UAST](https://doc.bblf.sh/uast/specification.html).
```

Comment:
I would add "Here is" at the start of the sentence or "these are simple instructions for...".

Reply:
how parse -> how to parse.